README.md

```diff
@@ -12,6 +12,7 @@
 * [Documentation](#documentation)
 * [License](#license)
 
+[![Build Status](https://travis-ci.org/thinkingfish/pelikan.svg?branch=travis_ci)](https://travis-ci.org/thinkingfish/pelikan)
```

Comment:
Certainly. This is just for me to see the effect of the icon on my own README.md, since the twitter repo doesn't have travis enabled yet. I will need admin privilege to add that (and I haven't got that approved by OSS yet)